ogr/services/mock/core.py

```diff
@@ -0,0 +1,82 @@
+import os
+import inspect
+import yaml
+
+def write_output(*args):
+    print(*args)
+
+class CallLogger(object):
+    def __init__(self, namespace, class_obj):
+        self.namespace = namespace
+        self.class_obj = class_obj
```

Comment:
I am not sure if I got the purpose of this class.

If it is for easier logging, why we have `namespace` attribute here -- it's specific to the `GithubProject/`PagureProject` classes.

Reply:
yes, the purpose is to see difference in log then as you mentioned `Github/Pagure`